openApi/openapi.yaml

```diff
@@ -264,9 +264,11 @@ paths:
       operationId: setWalletInstanceStatus
       requestBody:
         content:
```

Comment:
it looks like there are lots of unrelated changes to dictionary here, can we limit this PR to the changes of it.json ?